src/base/memory_monitor_sr.hpp

```diff
@@ -37,7 +37,7 @@
  * 2: debug tracking error
  * 3: debug all
  */
-#define MMON_DEBUG_LEVEL 0
+#define MMON_DEBUG_LEVEL 1
```

Comment:
Revert to 0 after getting result of our performace test.

Reply:
Can we set the default to 0 and change it at compile time using the -D option?